modules/serverless-pingsource.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-kn-source.adoc
+
+[id="serverless-pingsource_{context}"]
+= Using PingSource
+
+PingSource is used to periodically send ping events with a constant payload to an event consumer. It can be used to schedule sending events, like a timer, as shown in the example:
+
+.Example PingSource YAML
+----
+apiVersion: sources.knative.dev/v1alpha1
```

Comment:
the 0.13.0 release has both:  v1alpha1 _and_ v1alpha2 -  I'd recommend going w/ the later

```
apiVersion: sources.knative.dev/v1alpha1
kind: PingSource
metadata:
  name: test-ping-source
spec:
  schedule: "*/2 * * * *" <1>
  data: '{"message": "Hello world!"}' <2>
  sink: <3>
    ref:
      apiVersion: serving.knative.dev/v1
      kind: Service
      name: event-display
```

Reply:
e.g. like here:  https://github.com/matzew/knative-eventing-samples/blob/master/03-source_broker/020-ping-source.yaml

Reply:
please note that my demo sinks to a broker - instead of a service.  that's not important here.

but the rest shows the actual `v1alpah2` stuff